packages/grafana-e2e/src/support/types.ts

```diff
@@ -95,8 +101,11 @@ const processSelectors = <S extends Selectors>(e2eObjects: E2EFunctions<S>, sele
 
         // the input can only be (text, options)
         if (arguments.length === 2) {
-          const ariaText = value(textOrOptions as string);
-          const selector = Selector.fromAriaLabel(ariaText);
+          const text = textOrOptions as string;
```

Comment:
I know this is from before, but would it be possible to not cast it here and to handle the types correctly?